README.md

```diff
@@ -1,4 +1,9 @@
 # Henge
+
+## This project is not maintained anymore.
+## We joined  effords with [Kompose](https://github.com/skippbox/kompose) project.
+
```

Comment:
I think we should remove "not maintained anymore", since it *still* is, but it's just in the Kompose project :)

I would personally say:
```
# NOTICE
This project has merged into [Kompose](https://github.com/skippbox/kompose).

Please refer to [github.com/skippbox/kompose](https://github.com/skippbox/kompose) in order to make contributions to the further development of "Henge".

For a quick introduction to **Kompose** as well as a brief overview, please refer to [this blog post](http://www.skippbox.com/kompose/).
```